src/main/java/com/ssbb/kanban/dao/impl/ProjectDAO.java

```diff
@@ -8,14 +8,20 @@
 
 public class ProjectDAO extends DAOImpl<Storable> {
 
-	private final static Class<Project> projectClass = Project.class;
-	
-	public Project getProjectByName(String name) {
-		Query q = em.createNativeQuery(
-				"SELECT * FROM PROJECTS WHERE NAME = :name", projectClass);
-		q.setParameter("name", name);
-		
-		return (Project) q.getSingleResult();
-	}
+    private final static Class<Project> projectClass = Project.class;

```

Comment:
Just curios, did you copy and paste this existing code?  Since it seems to have registered a deletion of what I added.  It looks to me like you created your new branch off an older version of develop and added the same method manually.